apidoc/Titanium/Contacts/Contacts.yml

```diff
@@ -247,6 +247,21 @@ methods:
         type: Callback<ContactsAuthorizationResponse>
 
 properties:
+  - name: includeNote
+    summary: A boolean value that indicates whether to fetch the notes stored in contacts or not.
+    description:  |
+        This  property need to be set before calling contacts APIs.
+        From iOS 13 or later if your app fetch note field from contact, the app must have to set key 'com.apple.developer.contacts.notes' to 'true' in entitelements section of tiapp.xml.
+        
+        <key>com.apple.developer.contacts.notes</key>
+        <true/>
+        
+        See more details at https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_contacts_notes and https://developer.apple.com/contact/request/contact-note-field.
+    type: Boolean
+    default: true
+    platforms: [iphone, ipad]
+    since: 8.2.1
```

Comment:
From an architecture point of view, this is the wrong API design I think. Apple will very likely add more properties to the "sensitive" ones, so probably the user should have a property to override all fields that should be fetched - with the default not containing this one anymore. And it's likely a breaking change, just like this one also is.

Reply:
I can be agree with you that we have to change/extend the behavior of exiting APIs specially which are fetching use contacts data. Probably android also have to make some changes to have a parity and there may be some breaking change. For that I think we can not make it before 9.0.0. 
But this new property will  give developer an option to avoid fetching the 'note' in iOS otherwise they have to make some unnecessary setup. So there is no problem in having this new property. And by default this property is true, so there is no breaking change.

Reply:
Without implementing this property, apps are being rejected, so it's a breaking change for them. Either adding this property or defining the properties they actually need, which will be more future proof in the first place. But it's just a community comment 🙂